tests/python/contrib/test_ethosn/infrastructure.py

```diff
@@ -334,4 +334,4 @@ def get_ethosn_api_version():
 
 
 def get_ethosn_variant():
-    return os.getenv("ETHOSN_VARIANT_CONFIG", default="Ethos-N77")
+    return os.getenv("ETHOSN_VARIANT_CONFIG", default="Ethos-N78")
```

Comment:
Does this need to be the full variant descriptor? And can we remove the NPU specific block from `test_python_integration.sh` if this is the default?